client/src/App.js

```diff
@@ -19,28 +19,23 @@ function App() {
   function handleLogin(value) {
     setUsername(value);
   }
-  const CustomLink = styled(Link)`
-    text-decoration: none;
-  `;
+
   return (
     <UserContext.Provider value={username}>
       <ThemeProvider theme={normal}>
         <Router>
           <GlobalStyles />
-          <CustomLink to="/">
+          <LinkNoDeco to="/">
             <Header />
-          </CustomLink>
+          </LinkNoDeco>
           <Switch>
             <Route exact path="/">
               <LoginPage auth={handleLogin} />
             </Route>
-            <Route exact path="/overview">
-              <Main />
-            </Route>
-            <Route exact path="/addEvent">
-              <AddEvent />
-            </Route>
-            <Route path="/eventDetails">
+            <Route exact path="/overview" component={Main} />
+            <Route exact path="/addEvent" component={AddEvent} />
+            {/* <Route exact path="/eventDetails/:EventID" component={SearchPage} /> */}
```

Comment:
Do you still need this comment? Otherwise please delete it :)!

Reply:
I'll delete it. :)